spyderlib/tour.py

```diff
@@ -108,41 +118,72 @@ def get_tour(index):
-                           "into the tour."),
+    intro = [{'title': _("Welcome to the Introduction tour"),
+              'content': _("<b>Spyder</b> is a powerful Interactive "
+                           "Development Environment (or IDE) for the Python "
+                           "programming language.<br><br>"
+                           "Here we are going to guide you through its most "
+                           "important features.<br><br>"
+                           "Please use the arrow keys or click on the buttons "
+                           "below to move across the tour."),
               'image': 'tour-spyder-logo.png'},
 
              {'title': _("The Editor"),
-              'content': _("Decoration here is used to highlight the "
-                           "<b>Line Number Area</b> "
-                           "<br><br> No interaction example."),
+              'content': _("This is the pane where you write code. It comes "
+                           "with a line number area (highlighted here in red), "
+                           "where Spyder shows warnings and syntax errors that "
+                           "help you to detect potential problems before "
+                           "running the code.<br><br>"
+                           "You can also set debug breakpoints in this area "
+                           "by doing a double click next to a non-empty line."),
               'widgets': [sw.editor],
               'decoration': [sw.editor_line_number_area]},
 
              {'title': _("The IPython console"),
-              'content': _("Now lets try to run some code to show the nice "
-                           "things in <b>Spyder</b>.<br><br>"
-                           "Click when ready and pay close attention to the "
-                           "variable explorer"),
-              'widgets': [sw.console_ipython],
-              'run': ['a = 2', 'b = 4.0']
+              'content': _("This is one of panes where you can run or "
+                           "execute your code. This console comes with several "
+                           "useful features that greatly improve your "
+                           "programming workflow (like syntax highlighting and "
+                           "inline plots). You can read in depth about them "
+                           "<a href=\"%s\">here</a>.<br><br>"
+                           "It's also important to note that you can open as "
+                           "many consoles as you want, by going to the "
+                           "menu entry<br><br>"
+                           "<i>Consoles > Open an IPython console</i>.<br><br>"
+                           "Please click on the button below to run some simple "
+                           "code in this console. This will be useful to show "
+                           "you other important Spyder features." % \
+                           qtconsole_link),
+              'widgets': [sw.ipython_console],

```

Comment:
Yes, we need to verify that `ipython_console` exists before adding this part to the tour.